docs/tools/erc725-tools.md

```diff
@@ -6,3 +6,9 @@ sidebar_position: 6
 # ERC725 Tools
 
 - [ERC725 Inspect 📝🔍](https://erc725-inspect.lukso.tech/) ([GitHub](https://github.com/lukso-network/tools-erc725-inspect)) is a web app that allows you to inspect the key/values of an [ERC725Y](https://github.com/ERC725Alliance/ERC725/blob/main/docs/ERC-725.md#erc725x) compatible smart contract.
+
+<img
```

Comment:
Can you use align center maybe? Check in the repo how it was done. It is to center the image

Reply:
added now